packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -77,11 +88,30 @@ Because one Ethereum address is generally used many times to receive and send tr
 
 ### ERC20
 
-ERC20 tokens are based on Ethereum and therefore also use Ethereum addresses.
-We allow the creation of ERC20 requests with *proxy contract* payment network. *Input data* can't be specified for ERC20 transfers.
+Request is compatible with every ERC20 currency, but some of them have to be detailed manually. We use the package `eth-contract-metadata` to automatically fetch smart contracts and currency code of main currencies, whose the list is maintained by Metamask.
+
+Adding an additional ERC20 currency is very easy:
```

Comment:
Should we add first an example with an ERC20 from the list we already know ?

Reply:
I agree, we should show the easy use-case before showing the hard one.

Reply:
An example:
```typescript
const request = await requestNetwork.createRequest({
  paymentNetwork,
  requestInfo: {
    currency: 'DAI',
    expectedAmount: '1000000000000000000',
    payee: payeeIdentity,
    payer: payerIdentity,
  },
  signer: payeeIdentity,
});
```